### PR TITLE
Death sequence (death window)

### PR DESCRIPTION
**Pull request by kennedyford**

### CODE REVIEW

File: Project/Fall2020_CSC403_Project/FrmDeath.cs [hunk 1]
```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Windows.Forms;
+
+namespace Fall2020_CSC403_Project
+{
+    public partial class FrmDeath : Form
+    {
+        private Button Try_Again;
+        private Button Quit;
+        private Label label1;
```
Comment (joshuab112) [inline Project/Fall2020_CSC403_Project/FrmDeath.cs:10]: Maybe try adding a more specific name to this variable. What exactly does label1 display?

**Review by joshuab112 — COMMENTED** [review]
Overall it looks really good! My only suggestion would be give a more specific name to the Label in FrmDeath.cs

File: Project/Fall2020_CSC403_Project/FrmBattle.cs [hunk 2]
```diff
@@ -70,7 +71,15 @@ public partial class FrmBattle : Form {
       }
 
       UpdateHealthBars();
-      if (player.Health <= 0 || enemy.Health <= 0) {
+      if (player.Health <= 0) {
```
Comment (Brenbrit) [inline Project/Fall2020_CSC403_Project/FrmBattle.cs:77]: The player may also die while failing to flee. If you fetch changes from main, you can paste the logic from this statement in the flee button's function.

**Review by Brenbrit — CHANGES_REQUESTED** [review]
Would you mind also adding the death sequence statements for if a player dies in a failed attempt to flee? You'd have to fetch from main to get the flee button logic

File: Project/Fall2020_CSC403_Project/FrmDeath.cs (hunk 1 above)
Comment (kennedyford) [reply Project/Fall2020_CSC403_Project/FrmDeath.cs:10]: label1 is just the text that I put on the death window. I didn't even realize that it auto-created all of these code.

**Pull request merged**